composer.json

```diff
@@ -16,7 +16,7 @@
 
     "require": {
         "php":">=5.4",
-        "symfony/framework-bundle": "^2.4|^3.0|^4.0",
+        "symfony/framework-bundle": "4.0.*",
```

Comment:
We should still be compatible with Symfony 2.x and 3.x
`^2.4|^3.0|^4.0` will be the right constraint.